### ISSUE TITLE
Allow linking concepts to external resources

### ISSUE THREAD

**Issue by strogonoff** [glossarist/glossarist-desktop#140]
Requirements (as discussed with @ronaldtse)

* It is OK to restrict external resources to URIs only
* This feature may use same (or similar) relationship types as in normal (internal) relationships
* Fetch any structured data from the URI is not in scope for now

We may or may not use [the Relationships panel](https://www.glossarist.org/docs/desktop/ui/panels/relationships/) for this.

I believe we should restrict the Relationships panel to Glossarist-managed concepts only, and generic external resources should be implemented on another level.

**Comment by strogonoff** [comment]
@ronaldtse is an external linked resource specific to a language-specific entry, or to concept as a whole? Can it be that English entry refers to some external resources, and Spanish entry refers to other resources?

**Comment by ronaldtse** [comment]
@strogonoff good call. If we talk about SKOS, it is only about concept relationships. However, when we talk about designations and definitions, they can have other types of relationships (e.g. ISO authoritative source of definition). Can we list them out?

**Comment by strogonoff** [comment]
Per discussion outside Github, we might (for now) support external relationships on both levels, for top-level meaning units and for localized concept entries.

In future we will have top-level concepts and each language’s concept entries reside in their own registers, in which case we can have generic relationships on any concept item in concept register.

(NB: I believe designations are not supposed to have relationships at all. Concepts refer to units of meaning, while designations describe grammatical properties of words used to refer to concepts.)

**Comment by strogonoff** [comment]
@ronaldtse do you think external references may be used for clarifying e.g. why a certain word was chosen to designate a certain concept?

If those exist, we might want to support external relationships for _designations_ as well

**Comment by strogonoff** [comment]
Up until now, relationships are supposed to only be pertaining to definitions/meaning, not designations

**Comment by ronaldtse** [comment]
> (NB: I believe designations are not supposed to have relationships at all. Concepts refer to units of meaning, while designations describe grammatical properties of words used to refer to concepts.

I wonder if designations should be allowed to have relationships with other designations. i.e. this "designation comes from this external source".

**Comment by ronaldtse** [comment]
We should first allow linking of concepts and the consider designations. The former is critical for the importing of SKOS #127 .
